src/test/java/com/google/api/generator/gapic/composer/GrpcServiceStubClassComposerTest.java

```diff
@@ -55,7 +55,21 @@ public void generateServiceClasses() {
 
   // TODO(miraleung): Update this when a file-diffing test mechanism is in place.
   private static final String EXPECTED_CLASS_STRING =
-      "package com.google.showcase.v1beta1.stub;\n"
+      "// Copyright 2020 Google LLC\n"
```

Comment:
We can remove these blocks when we migrate the header addition to Composer.java.

Reply:
It's removed since it's tested in `ComposerTest`